spring-data-jdbc/src/main/java/org/springframework/data/jdbc/repository/query/AbstractJdbcQuery.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2020 the original author or authors.
+ * Copyright 2021 the original author or authors.
```

Comment:
I'm just mentioning this because you obviously put quite some effort in adhering to all the rules (Thanks for that).

Copyright should be `2000-2021`.